src/last10.c

```diff
@@ -54,18 +52,9 @@ int last10(void) {
 	}
     }
 
-    /* index points to the first QSO */
+    /* index points to the 10th QSO */
     if (index < 0)
-	return (-1);			/* not 10 QSOs found */
+	return -1;			/* not 10 QSOs found */
 
-    minsbefore = qso->hour *60 + qso->min;
-
-    minsnow = get_minutes();
-
-    if ((minsnow - minsbefore) <= 0)
-	minsnow += 1440;
-
-    span = minsnow - minsbefore;
-
-    return span;
+    return (get_time() - qso->timestamp) / 60;  /* elapsed time in minutes */
```

Comment:
Good idea. I like it. Not only solves it the >24h problem but is also more efficient.